<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>My Solar System</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>693</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Simulations">
     <property name="title">
      <string>Recent Simulations</string>
     </property>
     <addaction name="actionTest_xml"/>
    </widget>
    <addaction name="actionNew_Simulation"/>
    <addaction name="actionOpen"/>
    <addaction name="menuRecent_Simulations"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionNew_Object"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionPrefenrences"/>
   </widget>
   <widget class="QMenu" name="menuAbout_MySolarSystem">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_MySolarSystem"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuShow">
     <property name="title">
      <string>Show</string>
     </property>
     <addaction name="actionObject_tree"/>
    </widget>
    <addaction name="actionFull_screen"/>
    <addaction name="actionLeave_Full_Screen"/>
    <addaction name="separator"/>
    <addaction name="menuShow"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuAbout_MySolarSystem"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="uniformRowHeights">
        <bool>false</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <attribute name="headerDefaultSectionSize">
        <number>140</number>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Objects</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Values</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Sun</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>Mass</string>
         </property>
         <property name="text">
          <string>2000</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Earth</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>Mass</string>
         </property>
         <property name="text">
          <string>200</string>
         </property>
        </item>
       </item>
       <item>
        <property name="text">
         <string>Moon</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <item>
         <property name="text">
          <string>Mass</string>
         </property>
         <property name="text">
          <string>10</string>
         </property>
        </item>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_5">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Time unit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>23240245</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionNew_Simulation">
   <property name="text">
    <string>New...</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionTest_xml">
   <property name="text">
    <string>test.xml</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPrefenrences">
   <property name="text">
    <string>Prefenrences</string>
   </property>
  </action>
  <action name="actionAbout_MySolarSystem">
   <property name="text">
    <string>About MySolarSystem</string>
   </property>
  </action>
  <action name="actionNew_Object">
   <property name="text">
    <string>New Object</string>
   </property>
  </action>
  <action name="actionFull_screen">
   <property name="text">
    <string>Full Screen</string>
   </property>
  </action>
  <action name="actionLeave_Full_Screen">
   <property name="text">
    <string>Leave Full Screen</string>
   </property>
  </action>
  <action name="actionObject_tree">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Object tree</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew_Object</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFull_screen</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLeave_Full_Screen</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionObject_tree</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockWidget_2</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>531</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
